optaweb-employee-rostering-benchmark/src/test/java/org/optaweb/employeerostering/BenchmarkTest.java

```diff
@@ -49,6 +50,8 @@ public void isBenchmarkReportGeneratedTest() throws FileNotFoundException {
                 .filter(f -> !oldBenchmarkFilesInDirectory.contains(f))
                 .findAny()
                 .orElseThrow(() -> new FileNotFoundException("No benchmark report found"));
-        Assertions.assertTrue(benchmarkReport.exists());
+        assertThat(benchmarkReport.exists())
+                .withFailMessage("No Benchmark Report at " + benchmarkReport.getAbsolutePath())
+                .isTrue();
```

Comment:
You can use `assertThat(benchmarkReport).exists()`. Always prefer specialized assertions because they give useful failure descriptions out of the box.